java/src/jmri/LightManager.java

```diff
@@ -203,4 +203,9 @@
     @CheckReturnValue
     public boolean allowMultipleAdditions(@Nonnull String systemName);
 
+    /**
+     * Provide a connection specific tooltip and entry validation regex for the Add new item beantable pane.
+     */
+    public String[] getAddFormat();
```

Comment:
I used 2 separate methods at first, but as these items are are both plain strings and used (only) side by side through a call to the same manager, that seems less efficient. Both are GUI related, and work together in just 1 place, on the pane (until we force the mask to every entry. It's too early for that). But we certainly should think of alternative approaches, no problem. Are concerned about performance or OOP principles? Glad the tests passed.